openupgrade_scripts/scripts/website_slides_survey/16.0.1.0/pre-migration.py

```diff
@@ -0,0 +1,50 @@
+# Copyright 2025 Tecnativa - Carlos Lopez
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openupgradelib import openupgrade
+
+_translations_to_delete = [
```

Comment:
Being strict, these are used as constants, so they can be put in upper and with a meaningful name, but you can pass directly the value in the method call instead of putting it here.

```suggestion
XMLIDS_TO_DELETE_TRANSLATIONS = [
```